packages/jest-config/package.json

```diff
@@ -9,6 +9,14 @@
   "license": "MIT",
   "main": "build/index.js",
   "types": "build/index.d.ts",
+  "peerDependencies": {
+    "ts-node": "^9.0.0"
```

Comment:
Should probably be a dev dep as well - I think PnP would have complained about it missing if we'd been using it